web/src/compose_tooltips.ts

```diff
@@ -35,7 +36,7 @@ export function initialize(): void {
         },
     });
     tippy.delegate("body", {
-        target: "#compose_buttons .compose-reply-button-wrapper",
+        target: ".compose-reply-button-wrapper",
```

Comment:
Why does this selector need to change?

Reply:
Fixed that. Might have been removed by accident.